js/26_09_19.test.js

```diff
@@ -0,0 +1,10 @@
+const likes = require('./26_09_19');
+
+test('4 likes', () => {
```

Comment:
When creating multiple tests for a given function/module, use a `describe` block to group them

https://jestjs.io/docs/en/api#describename-fn

Reply:
Done